src/clients/Device.js

```diff
@@ -811,15 +813,13 @@ class Device extends EventEmitter {
   };
 
   releaseOwnership = (flasher: Flasher) => {
-    logger.log('releasing flash ownership ', { deviceID: this.getDeviceID() });
+    logger.info({ deviceID: this.getDeviceID() }, 'releasing flash ownership ');
     if (this._owningFlasher === flasher) {
       this._owningFlasher = null;
     } else if (this._owningFlasher) {
       logger.error(
-        'cannot releaseOwnership, ',
-        flasher,
-        " isn't the current owner ",
-        { deviceID: this.getDeviceID() },
+        { deviceID: this.getDeviceID(), flasher },
+        'cannot releaseOwnership, isn\'t  current owner'
```

Comment:
there should be trailing comma here, so you defenitly didn't run percommit hooks for the pr.